app/commands/codemod/confirm.go

```diff
@@ -0,0 +1,157 @@
+package codemod
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/codelingo/lingo/app/util"
+	flow "github.com/codelingo/platform/flow/rpc/flowengine"
+	"github.com/fatih/color"
+	"github.com/waigani/diffparser"
+)
+
+type IssueConfirmer struct {
+	keepAll bool
+	output  bool
+}
+
+func NewConfirmer(output, keepAll bool, d *diffparser.Diff) (*IssueConfirmer, error) {
+	cfm := IssueConfirmer{
+		keepAll: keepAll,
+		output:  output,
+	}
+
+	return &cfm, nil
+}
+
+func GetDiffRootPath(filename string) string {
+	// Get filename relative to git root folder
+	// TODO: Handle error in case of git not being installed
+	// https://github.com/codelingo/demo/issues/2
+	out, err := exec.Command("git", "ls-tree", "--full-name", "--name-only", "HEAD", filename).Output()
+	if err == nil && len(out) != 0 {
+		if len(out) != 0 {
+			filename = strings.TrimSuffix(string(out), "\n")
+		}
+	}
+	return filename
+}
+
+var editor string
+
```

Comment:
comment on exported method IssueConfirmer.Confirm should be of the form "Confirm ..."